src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3670/CREATE_EOF1InvalidOpcodesFiller.yml

```diff
@@ -0,0 +1,176 @@
+# Fails to deploy EOF code containing each one of Invalid Opcodes
+CREATE_EOF1InvalidOpcodes:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul
+        {
+          // code to deploy: <invalid_opcode> STOP
+          // initcode: { mstore(0, 0xef00010100040200010002030000000000000000000000000000000000000000) return(0, 21) }
+          mstore(128, 0x7fef000101000402000100020300000000000000000000000000000000000000)
+          mstore(160, 0x0060005260156000f30000000000000000000000000000000000000000000000)
+
+          // Add the opcode to the init code
+          let code := shr(248, calldataload(0))
+          mstore8(148, code)
+
+          // Try to deploy EOF code containing invalid opcode
+          if create(0, 128, 41) { sstore(code, 1) }
+        }
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data: # Send invalid opcodes as data
+      - '0x0c'
+      - '0x0d'
+      - '0x0e'
+      - '0x0f'
+      - '0x1e'
+      - '0x1f'
+      - '0x21'
+      - '0x22'
+      - '0x23'
+      - '0x24'
+      - '0x25'
+      - '0x26'
+      - '0x27'
+      - '0x28'
+      - '0x29'
+      - '0x2a'
+      - '0x2b'
+      - '0x2c'
+      - '0x2d'
+      - '0x2e'
+      - '0x2f'
+      - '0x49'
+      - '0x4a'
+      - '0x4b'
+      - '0x4c'
+      - '0x4d'
+      - '0x4e'
+      - '0x4f'
+      - '0x56' # JUMP deprecated in EIP-4750
+      - '0x57' # JUMPI deprecated in EIP-4750
```

Comment:
``suggestion
      - '0x57' # JUMPI deprecated in EIP-4750
      - '0x58' # PC deprecated in EIP-4750
```